docs/docs/how-to/sourcing-data/sourcing-from-the-filesystem.md

```diff
@@ -96,6 +96,31 @@ You can then update your query using `sourceInstanceName` and the value of the `
 }
 ```
 
+## Sourcing Files using Environment Variables
```

Comment:
```suggestion
## Conditionally sourcing files using environment variables
```

One is not really "using" environment variables but is using them to conditionally set `path`